src/components/select/template.njk

```diff
@@ -0,0 +1,18 @@
+{% from "label/macro.njk" import govukLabel %}
+{% set labelArguments = {
+  text: params.label.text if params.label.text else params.label,
```

Comment:
This won't work if `html` is set instead of `text`, as `params.label.text` will be `nil` and so the `params.label` object will be passed through as `text`.

Reply:
in the second example, i set html on the label, which seems to work fine

Reply:
So it does. I guess that works because `html` is set so we never output `text`.

However this still feels a little hacky to me – whilst it might work we're still doing odd things like dumping the whole label object into text, even if it's not used.